### DISCUSSION TITLE
Sametime, defaults for advanced settings?

### DISCUSSION THREAD

**Discussion by paulswithers** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#13] [Ideas]
For Sametime we have to have slight differences. And we wanted to avoid an additional dialog for advanced settings, so we just set the basics.

For Sametime you can require a password and allow guests. Would it be preferable to allow that to be set on the online meeting credential for Sametime, as a default if you want to set those to true?

I want to avoid using notes.ini variables, so we don't add extra difficulties that would make Verse have to work differently.

(I'm leaving aside the other meeting providers for the moment, just to keep the thread clean.)

**Comment by rboccadoro** [comment]
Yes, allowing guest would be needed. Often my customers do meeting with suppliers/partners/....
